pkg/bindings/generate/generate.go

```diff
@@ -31,38 +30,3 @@ func Systemd(ctx context.Context, nameOrID string, options *SystemdOptions) (*en
 	report := &entities.GenerateSystemdReport{}
 	return report, response.Process(&report.Units)
 }
-
-// Kube generate Kubernetes YAML (v1 specification)
-//
-// Note: Caller is responsible for closing returned reader
-func Kube(ctx context.Context, nameOrIDs []string, options *KubeOptions) (*entities.GenerateKubeReport, error) {
```

Comment:
Would it not be a breaking change if this calls `kube.Generate()`? I have kept the Generate() and Kube() functions around just moved them around a bit between packages and made sure that the old functions stay and call the new ones, the only thing I forgot is to maintain this `Kube()` function and call `kube.Generate()` here instead. We already did this with PlayKube/KubePlay and I didn't think this would be a breaking change if I preserved the old functions and just called the new ones in them.

I might be mistaken, please let me know and I will be happy to drop the commit. I would just like to understand a bit more first.

Reply:
I fixed the `Kube()` function here to call `kube.Generate()` - PTAL and let me know if it is still considered a breaking change.

Reply:
@umohnani8 This is still a breaking change because you changed the types.
If anybody imports that code it will fail to compile after the update.

At the very least you have to type alias them if we need to keep backwards compat: https://yourbasic.org/golang/type-alias/

Reply:
Ah, I see. Thanks for the pointer. Adding a type alias for KubeOptions is making go generate unhappy. So in the interest of time, I have dropped that commit for now and will open another PR once this is on for that.